step1-clean/db.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine, Column, String, Integer
+
+Base = declarative_base()
+
+class SQLImage(Base):
+  __tablename__ = 'Thumbnails'
+
+  id = Column(Integer, primary_key=True)
+  filename = Column(String)
+  isThumbnail = Column(String)
```

Comment:
I would not store both thumbnails and images in the same table. The table is intended for managing the relationships between thumbnails and images i.e. find thumbnail for an image or find image from a thumbnail

Reply:
Understood. I thought this was a bit strange, but dismissed it because the instructions seemed to say add both to the same table. I should have asked. 

Fixed.